Patches/T's Samurai Faction/T_Samurai_Weapons_Ranged.xml

```diff
@@ -0,0 +1,387 @@
+            ]/tools</xpath>
+          <value>
+          <tools>
+          <li Class="CombatExtended.ToolCE">
+            <label>stock</label>
+            <capacities>
+            <li>Blunt</li>
+            </capacities>
+            <power>8</power>
+            <cooldownTime>1.55</cooldownTime>
+            <chanceFactor>1.5</chanceFactor>
+            <armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+            <linkedBodyPartsGroup>Stock</linkedBodyPartsGroup>
+          </li>
+          <li Class="CombatExtended.ToolCE">
+            <label>barrel</label>
+            <capacities>
+            <li>Blunt</li>
+            </capacities>
+            <power>5</power>
+            <cooldownTime>2.02</cooldownTime>
+            <armorPenetrationBlunt>1.630</armorPenetrationBlunt>
+            <linkedBodyPartsGroup>Barrel</linkedBodyPartsGroup>
+          </li>
+          <li Class="CombatExtended.ToolCE">
+            <label>muzzle</label>
+            <capacities>
+            <li>Poke</li>
+            </capacities>
+            <power>8</power>
+            <cooldownTime>1.55</cooldownTime>
+            <armorPenetrationBlunt>2.755</armorPenetrationBlunt>
+            <linkedBodyPartsGroup>Muzzle</linkedBodyPartsGroup>
+          </li>
+          </tools>
+          </value>
+        </li>
+
+        <!-- Short Gun -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+            <defName>TSF_Tanegashima_Short</defName>
+            <statBases>
+              <WorkToMake>12000</WorkToMake>
+              <Bulk>4.75</Bulk>
+              <Mass>2.5</Mass>
+              <SwayFactor>1.22</SwayFactor>
+              <ShotSpread>0.30</ShotSpread>
+              <SightsEfficiency>0.7</SightsEfficiency>
+              <RangedWeapon_Cooldown>0.85</RangedWeapon_Cooldown>
+            </statBases>
+            <costList>
+              <Steel>35</Steel>
+              <WoodLog>8</WoodLog>
+              <ComponentIndustrial>1</ComponentIndustrial>
+            </costList>
+            <Properties>
+              <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+              <hasStandardCommand>True</hasStandardCommand>
+              <defaultProjectile>Bullet_SlowMusketBall</defaultProjectile>
+              <warmupTime>1.1</warmupTime>
+              <range>24</range>
+              <soundCast>Shot_Musket</soundCast>
+              <soundCastTail>GunTail_Light</soundCastTail>
+              <muzzleFlashScale>9</muzzleFlashScale>
+            </Properties>
+            <AmmoUser>
+              <magazineSize>1</magazineSize>
+              <reloadTime>7</reloadTime>
+              <ammoSet>AmmoSet_SlowMusketBall</ammoSet>
+            </AmmoUser>
+            <FireModes>
+              <aiAimMode>AimedShot</aiAimMode>
+            </FireModes>
+            <weaponTags>
+              <li>NeolithicRangedHeavy</li>
+              <li>NeolithicRangedChief</li>
+              <li>CE_OneHandedWeapon</li>
```

Comment:
I'd remove this tag, as the weapon really isn't manageably with a shield.

Reply:
wierd, I remember having already removed that